components/brave_wallet/browser/json_rpc_service.cc

```diff
@@ -166,9 +166,22 @@ void JsonRpcService::AddObserver(
 void JsonRpcService::Request(const std::string& json_payload,
                              bool auto_retry_on_network_change,
                              mojom::CoinType coin,
+                             const std::string& chain_id,
                              RequestCallback callback) {
-  RequestInternal(json_payload, auto_retry_on_network_change,
-                  network_urls_[coin], std::move(callback));
+  auto network_url = GetNetworkURL(prefs_, chain_id, coin);
+  if (!network_url.is_valid()) {
+    return;
```

Comment:
BTW you can't just return for this, error needs to be handled, callback needs to be called.

Reply:
I believe this is fixed now